cime_config/testdefs/testlist_cam.xml

```diff
@@ -43,6 +43,15 @@
       <option name="comment">Test for Moist Held-Suarez (TJ2016) physics (physics_after_coupler group)</option>
     </options>
   </test>
+  <test compset="FPHYStest" grid="ne3pg3_ne3pg3_mg37" name="SMS_Ln2" testmods="cam/outfrq_zm_derecho">
```

Comment:
oops! missed this during my review, but I believe you need to commit the new outfrq_zm_derecho directory.

Reply:
Will need to get this from derecho once it is back up

Reply:
Added